ci/Dockerfile

```diff
@@ -1,9 +1,96 @@
-FROM a12e/docker-qt:5.14-gcc_64
+FROM ubuntu:18.04
+
+# Adapted from a12e/docker-qt by 
+
+ARG OPENSSL_VERSION=1.1.1d
+ARG QT_VERSION=5.15.2
+ARG SDL_VERSION=2.0.10
+ARG LINUXDEPLOYQT_VERSION=continuous
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    OPENSSL_PREFIX=/usr/local/ssl \
+    QMAKESPEC=linux-g++ \
+    QT_PATH=/opt/qt \
+    QT_PLATFORM=gcc_64 \
+    QT_VERSION=$QT_VERSION
+
+ENV \
+    PATH=${QT_PATH}/${QT_VERSION}/${QT_PLATFORM}/bin:${OPENSSL_PREFIX}/bin:$PATH
+
+# Install updates & requirements:
+#  * git, openssh-client, ca-certificates - clone & build
+#  * locales, sudo - useful to set utf-8 locale & sudo usage
+#  * curl - to download Qt bundle
+#  * build-essential, pkg-config, libgl1-mesa-dev - basic Qt build requirements
+#  * libsm6, libice6, libxext6, libxrender1, libfontconfig1, libdbus-1-3 - dependencies of the Qt bundle run-file
+#  * wget - another download utility
+#  * libz-dev - OpenSSL dependencies
+#  * fuse, file - linuxdeployqt dependencies
+#  * libxkbcommon-x11-0 - run-time dependencies
+#  * libgstreamer-plugins-base1.0-0 - QtMultimedia run-time dependencies
+RUN apt update && apt full-upgrade -y && apt install -y --no-install-recommends \
+    git \
+    openssh-client \
+    ca-certificates \
+    locales \
+    sudo \
+    curl \
+    build-essential \
+    pkg-config \
+    libgl1-mesa-dev \
+    libsm6 \
+    libice6 \
+    libxext6 \
+    libxrender1 \
+    libfontconfig1 \
+    libdbus-1-3 \
+    wget \
+    libz-dev \
+    fuse \
+    file \
+    libxkbcommon-x11-0 \
+    wget \
+    p7zip-full \
+    make build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncurses5-dev libncursesw5-dev xz-utils tk-dev \
+    libgstreamer-plugins-base1.0-0 \
+    python3 python3-pip \
+    && apt-get -qq clean
+
+COPY scripts/* /tmp/build/
+
+# Download and install OpenSSL
+RUN /tmp/build/install-openssl-linux.sh
+
+# Download & unpack Qt toolchain
+RUN /tmp/build/install-qt.sh
+
+
+# Download & install linuxdeployqt
+RUN /tmp/build/install-linuxdeployqt.sh
+
+# Reconfigure locale
+RUN locale-gen en_US.UTF-8 && dpkg-reconfigure locales \
+# Add group & user
+    && groupadd -r user && useradd --create-home --gid user user && echo 'user ALL=NOPASSWD: ALL' > /etc/sudoers.d/user
+
+
+RUN python3 -m pip install setuptools
+
+RUN chmod -R 777 /opt
+
+USER user
+WORKDIR /home/user
+ENV HOME /home/user
+
+# Install QT
+RUN python3 -m pip install -U pip
+RUN python3 -m pip install aqtinstall
+RUN python3 -m aqt install --output "$QT_PATH" ${QT_VERSION} linux desktop -m qtwebengine
```

Comment:
Actually, wouldn't it make more sense to cut things off here, and start a new Docker image, and just copy over what `aqt` has installed to the new image without any of the dependencies necessary to do the installation?

https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

Reply:
You can see an example of this in our `nim-waku` repo, but also in many others:
https://github.com/status-im/nim-waku/blob/master/Dockerfile

Reply:
I'll do that. I did not know that this was something possible to do.